b3lb/rest/models.py

```diff
@@ -750,8 +792,8 @@ class Parameter(Model):
     # Join Parameters
     # see https://docs.bigbluebutton.org/admin/customize.html#passing-custom-parameters-to-the-client-on-join for documentation
     #
-    # some join parameters needs settings.yml defined inputs, see
```

Comment:
Looks like a accidentally search-and-replace.

Reply:
That's an search and replace accident, will be fixed.